pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[project]
+name = "pythran"
+authors = [{name="Serge Guelton", email="serge.guelton@telecom-bretagne.eu"}]
+license = {file = "LICENSE"}
+description = "Ahead of Time compiler for numeric kernels"
+requires-python = ">=3.7"
+classifiers=[
+  'Development Status :: 4 - Beta',
+  'Environment :: Console',
+  'Intended Audience :: Developers',
+  'License :: OSI Approved :: BSD License',
+  'Natural Language :: English',
+  'Operating System :: POSIX :: Linux',
+  'Operating System :: MacOS',
+  'Programming Language :: Python :: 3',
+  'Programming Language :: Python :: Implementation :: CPython',
+  'Programming Language :: C++',
+  'Topic :: Software Development :: Compilers',
+  'Topic :: Software Development :: Code Generators'
+]
+dynamic = ["version", "readme", "dependencies"]
+
+[project.urls]
+Homepage = "https://github.com/serge-sans-paille/pythran"
+Documentation = "https://pythran.readthedocs.io"
+Changelog = "https://pythran.readthedocs.io/en/latest/Changelog.html"
+
+[project.optional-dependencies]
+doc = ["numpy", "nbsphinx", "scipy", "guzzle_sphinx_theme"]
+test = ["ipython", "nbval", "cython", "wheel"]
+
+[project.scripts]
+pythran = "pythran.run:run"
+pythran-config = "pythran.config:run"
+
+[build-system]
+requires = [
+    "ply>=3.4",
+    "setuptools>=61.0",
+    "gast~=0.5.0",
+    "numpy",
```

Comment:
I don't think `numpy` is needed? I can build a wheel just fine with `python -m build` or `pip install .`, and I am not aware of the content of that wheel then depending on whether `numpy` was available in the build env.

Reply:
This comment still applies I think. For the rest, this PR now LGTM.

Reply:
Numpy is a required runtime dependency of Pythran. Isn't that what's captured by that field?